### PR TITLE
Feature/display comments #14

### PR DESCRIPTION
**Pull request by blessedjasonmwanza**
## Feature/display comments #14
In this UX update:
- Create [comments module](https://github.com/blessedjasonmwanza/ecommerce-rating/blob/feature/display-comments-%2314/src/comments.js) and [image resource](https://github.com/blessedjasonmwanza/ecommerce-rating/blob/feature/display-comments-%2314/src/img/close_icon.png) with methods to display popup
- Add [config module](https://github.com/blessedjasonmwanza/ecommerce-rating/blob/feature/display-comments-%2314/src/config.js) to keep and store API keys and related values for universal access

### CODE REVIEW

**Review by ntazelo — CHANGES_REQUESTED** [review]
### Changes required ♻️ 
you did great work 👍  But there is an issue you still need to work on.

Kindly check the comments under the review.

File: src/comments.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    return response;
+  }
+
+  enable() {
+    const commentsBtns = document.querySelectorAll('.btn-comment');
+    if (commentsBtns) {
+      commentsBtns.forEach((comment) => {
+        comment.addEventListener('click', () => {
+          const id = comment.getAttribute('id');
+          this.display(id);
```
Comment (ntazelo) [inline src/comments.js:95]: when commentsBtns is clicked the comments are not displayed . Please fix that.

**Review by ntazelo — CHANGES_REQUESTED** [review]

File: src/comments.js (hunk 1 above)
Comment (blessedjasonmwanza) [reply src/comments.js:95]: Right on it. Thanks a lot 😃

Comment (blessedjasonmwanza) [reply src/comments.js:95]: Actioned ✅

**Review by ntazelo — APPROVED** [review]
### ### Status: Approved ✔️ ✔️ ✔️

Thank you for implementing changes suggested previously 🙏. That is nice work 👏

**Pull request merged**